tests/phpunit/test-wc-rest-stripe-account-keys-controller.php

```diff
@@ -0,0 +1,249 @@
+<?php
+/**
+ * These tests make assertions against class WC_REST_Stripe_Account_Keys_Controller.
+ *
+ * @package WooCommerce_Stripe/Tests/WC_REST_Stripe_Account_Keys_Controller
+ */
+
+/**
+ * WC_REST_Stripe_Account_Keys_Controller unit tests.
+ */
+class WC_REST_Stripe_Account_Keys_Controller_Test extends WP_UnitTestCase {
+	/**
+	 * Tested REST route.
+	 */
+	const ROUTE = '/wc/v3/wc_stripe/upe_flag_toggle';
+
+	/**
+	 * The system under test.
+	 *
+	 * @var WC_REST_Stripe_Account_Keys_Controller
+	 */
+	private $controller;
+
+	/**
+	 * Pre-test setup
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		require_once WC_STRIPE_PLUGIN_PATH . '/includes/admin/class-wc-rest-stripe-account-keys-controller.php';
```

Comment:
What do you think about moving this to `tests/bootstrap.php`?

Reply:
Updated!